lib/matplotlib/ticker.py

```diff
@@ -2502,64 +2654,119 @@ def view_limits(self, vmin, vmax):
         return result
 
 
-class LogitLocator(Locator):
+class LogitLocator(MaxNLocator):
     """
     Determine the tick locations for logit axes
     """
 
-    def __init__(self, minor=False):
-        """Place ticks on the logit locations."""
-        self.minor = minor
+    def __init__(self, minor=False, *, nbins="auto"):
+        """
+        Place ticks on the logit locations
 
-    def set_params(self, minor=None):
+        Parameters
+        ----------
+        nbins : int or 'auto', optional
+            Number of ticks. Only used if minor is False.
+        minor : bool, default: False
+            Indicate if this locator is for minor ticks or not.
+        """
+
+        self._minor = minor
+        MaxNLocator.__init__(self, nbins=nbins, steps=[1, 2, 5, 10])
+
+    @cbook._make_keyword_only("3.2", "minor")
```

Comment:
is there a reason for the API change here, and making `minor` a property?  (yes, I know, I suggested doing similar stuff in the docs PR, but that's for new APIs; we could also migrate older APIs to newer design but here it just lengthens the PR and makes it harder to review (unless there's a good reason for the change that I missed))

Reply:
As I rewrite competely the LogitLocator, I decide use `_minor` as private. I add the property after to have the compat with the previous locator. I do not know if it is a good reason.

Reply:
Do you actually want to deprecate `.minor`?  If so, that could be done in a separate PR.
(IOW the changes here are pretty orthogonal to the rest, so given the big size of the current PR, would benefit from being split out.)

Reply:
It is why I add a property without deprecate it, to easily add the deprecation on the getter and the setter without modifying the rest of the new code.

Reply:
I remove the deprecation of this args. Deprecation will be the topic of a future PR.